src/lib/i18n/dictionaries/englishDictionary.ts

```diff
@@ -1016,6 +1016,12 @@ const buildingDescriptions: Record<BuildingDescriptions, string> = {
   "description.tent": "(Discontinued)",
   "description.house": "A place to rest your head",
   "description.crop.machine": "Automate your crop production",
+  "building.oil.remaining": "{{oil}} Oil in building",
+
+  "cooking.building.oil.description":
+    "You can boost your recipe cooking times by {{boost}}% when adding oil into the {{buildingName}}. As you add oil you can see how long the {{buildingName}} will be able to be boosted.",
```

Comment:
I would suggest removing the second sentence. It doesn't add much context as the UI is already clear "As you add oil you can see how long the {{buildingName}} will be able to be boosted."